app/views/shared/_related_items.html+new_navigation.erb

```diff
@@ -1,3 +1,15 @@
 <div class="column-third add-title-margin">
-  <%= render partial: 'govuk_component/taxonomy_sidebar', locals: content_item.taxonomy_sidebar %>
+  <% if MainstreamContentFetcher.with_curated_sidebar.include?(content_item.base_path) %>
+    <%=
+      render(
+        partial: 'govuk_component/relted_items',
```

Comment:
there is no rendering going on locally, only a mock of the component with the data...